analysis_driver/client.py

```diff
@@ -61,8 +61,12 @@ def main(argv=None):
             scanner.report(all_datasets=True)
         return 0
 
-    datasets = scanner.scan_datasets(DATASET_NEW, DATASET_REPROCESS, DATASET_READY, DATASET_FORCE_READY)
-    ready_datasets = datasets.get(DATASET_FORCE_READY, []) + datasets.get(DATASET_READY, [])
+    processable_statuses = (DATASET_FORCE_READY, DATASET_READY, DATASET_RESUME)
```

Comment:
Can we swap `DATASET_READY` and `DATASET_RESUME` so that the resumed dataset take priority over the ready.